src/new-client/src/app/site/[site]/view-availability/week/page.tsx

```diff
@@ -0,0 +1,22 @@
+import NhsPage from '@components/nhs-page';
+import dayjs from 'dayjs';
+
+type PageProps = {
+  searchParams: {
+    from: string;
+    until: string;
```

Comment:
What's the need for the `until` parameter here? I'm just thinking that any searchParams are open to the users fudging them around. They could enter an `until` date that's a year away or in the past so we'll have to do some validation if we go down this route, at which point wouldn't it be simpler just to calculate the week from the week start date? 

Or go even further, and accept any single date and calculate the week even if it isn't the start? So if the user enters today's date that'll see 25th -> 1st? Should be doable with `date.startOf('isoWeek') and `date.endOf(isoWeek)`.

Reply:
It was just for the title of the new screen. Happy to update it but it's probably worth addressing in APPT-338 which covers this new page. I only added it in so the link went somewhere. I think I'll add TODOs in this file for now to be covered when either of us pick up APPT-338 :)